omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/DrugResource1_10.java

```diff
@@ -41,8 +47,10 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		DelegatingResourceDescription repDesc = super.getRepresentationDescription(rep);
 		if (rep instanceof DefaultRepresentation) {
 			repDesc.addProperty("strength");
+			repDesc.addProperty("drugReferenceMaps", Representation.REF);
```

Comment:
Is there any reason why you used drugReferenceMaps instead of drugReferenceMap?

Reply:
the Drug calss field is  `drugReferenceMaps;`
We dont have anything  like `drugReferenceMap`;